buildSrc/src/main/kotlin/org/cqfn/save/buildutils/DetektConfiguration.kt

```diff
@@ -21,6 +22,18 @@ fun Project.configureDetekt() {
         config = rootProject.files("detekt.yml")
         buildUponDefaultConfig = true
     }
+    // extremely awful hack to get to the point, when `DetektMultiplatform` has registered all the tasks
+    afterEvaluate {
+        afterEvaluate {
```

Comment:
Could you please provide little comments about why we need such number of `afterEvaluate` calls?